package/autoyast2.changes

```diff
@@ -1,4 +1,11 @@
 -------------------------------------------------------------------
+Wed Apr  5 10:34:17 CEST 2017 - schubi@suse.de
+
+- Added configuration-management to first installation step.
```

Comment:
np: I would rather prefer the word `stage` instead of `step`. But it's maybe a matter of taste...